site/src/pages/release-notes/1.1.0.mdx

```diff
@@ -0,0 +1,86 @@
+---
+date: 2020-09-18
+---
+
+## 🌟 New features
+
+- You can now inject your own <type://MeterIdPrefixFunction> when using Armeria Spring boot autoconfigure. #3034
+  ```java
+  @Bean
+  public MeterIdPrefixFunction myMeterIdPrefixFunction() {
+      return MeterIdPrefixFunction.of("custom.armeria.server");
+  }
+  ```
+- You can now use a string to specify the `Logger` for <type://LoggingService> and <type://LoggingClient>. #3035
+  ```java
+  ServerBuilder sb = ...
+  sb.decorator(LoggingService.builder()
+                             .logger("com.foo")
+                             .build());
+  ```
+
+## 📈 Improvements
+
+- Span name is the service name if you use the default `HttpRequestParser` and `HttpResponseParser` with
+  `BraveClient` and `BraveService`. #3023
+- You can now easily find out whether SPI service interfaces are applied from the logs. #3039
+- You can now get the log of the exception which is raised while making request and response logs. #3043
+- The stack-trace of <type://HttpStatusException> and <type://HttpResponseException> are not logged in
+  <type://LoggingService>. #3053, #3054
+
+## 🛠️ Bug fixes
+
+- <type://TimeoutScheduler> is initialized exactly once. #3066
+- Reactor subscriber context is now properly propagated. #3058
+- Armeria Spring boot autoconfigure collects metrics correctly if <type://ArmeriaSettings#enableMetrics>
+is `true`. #3048
+- You now see the proper exception from WebFlux `WebClient` when <type://CircuitBreakerClient> is applied. #3064
+- Armeria client rejects `PUSH_PROMISE` frame correctly because it doesn't support it yet. #3045
```

Comment:
- rejects -> now rejects
- a PUSH_PROMISE frame (missing a)

Reply:
A `PUSH_PROMISE` frame is not rejected. `SETTINGS_ENABLE_PUSH` of the initial settings is set to 0. 

Armeria client tells the server that it doesn't expect `PUSH_PROMISE`. ?

Reply:
> A PUSH_PROMISE frame is not rejected.

If a non Armeria server ignores `SETTINGS_ENABLE_PUSH` and sends `PUSH_PROMISE`, we don't reject the frame? then I think we should fix the code. 😄